TFT/src/User/API/Temperature.c

```diff
@@ -210,6 +214,14 @@ void loopCheckHeater(void)
       heat_update_waiting = true;
     } while (0);
   }
+  else // check temperature auto-report timout and resend M155 command
+  {
+    if (OS_GetTimeMs() > AUTOREPORT_TIMEOUT && !heat_update_waiting)
+    {
+      if(heat_update_waiting) updateNextHeatCheckTime();  // set next timeout for temperature auto-report
+      heat_update_waiting = storeCmd("M155 ");
```

Comment:
`heat_update_waiting = storeCmd("M155 ");`
`if(heat_update_waiting) updateNextHeatCheckTime();`

The order should be reversed like above, otherwise "if(heat_update_waiting) updateNextHeatCheckTime()" is never executed.

Reply:
🤦‍♂️it was supposed to be this way but somehow the lines got interchanged accidentally.